packages/app/src/app/pages/Dashboard/Components/Repo/RepoListItem.tsx

```diff
@@ -63,9 +76,21 @@ export const RepoListItem = ({
             </Stack>
           </Stack>
         </Column>
-        <Column span={[0, 4, 4]} as={Stack} align="center">
-          <Text size={3} weight="medium" variant="muted">
-            {props.owner}
+        <Column span={[0, 6, 1]} as={Stack} align="center">
+          <Text
+            size={3}
+            paddingX={4}
+            paddingY={2}
+            css={{
+              color: '#C2C2C2',
+              backgroundColor: '#1D1D1D',
+              borderRadius: 99,
+              ':hover, :focus, :focus-within': {
+                backgroundColor: '#303030',
```

Comment:
I would only change the backgroundColor on `hover` and add a `focus` outline with purple, like in the sandbox cards.

Reply:
This only changes the color of the Synced tab on hover so it doesn't fade in the background. The background color changes for the entire component on `hover` and `focus` had already been implemented  on line 46-48.